vars/parallelDockerUpdatecli.groovy

```diff
@@ -30,9 +42,14 @@ def call(userConfig = [:]) {
     },
     'updatecli': {
       withCredentials([string(credentialsId: finalConfig.credentialsId,variable: 'UPDATECLI_GITHUB_TOKEN')]) {
-        updatecli(action: 'diff', containerMemory: finalConfig.containerMemory)
+        // Merging the 2 maps - https://blog.mrhaki.com/2010/04/groovy-goodness-adding-maps-to-map_21.html
+        Map updatecliConfig = finalConfig.updatecliConfig << [action: 'diff']
+
+        updatecli(updatecliConfig)
         if (env.BRANCH_IS_PRIMARY) {
-          updatecli(action: 'apply', cronTriggerExpression: finalConfig.cronTriggerExpression, containerMemory: finalConfig.containerMemory)
+          // Merging the 2 maps - https://blog.mrhaki.com/2010/04/groovy-goodness-adding-maps-to-map_21.html
+          updatecliConfig = finalConfig.updatecliConfig << [action: 'apply', cronTriggerExpression: defaultUpdatecliCronTriggerExpression]
```

Comment:
I fail to see how the cronTriggerExpression can be changed by the user with this.
I would have expected here something like `defaultConfig.updatecliConfig.cronTriggerExpression` or better yet, something like `finalConfig.updatecliConfig.cronTriggerExpression`

Reply:
look at the test :) 
Joke out:
```
parallelDockerUpdatecli([
  image 'foo',
  updatecliConfig: [
    cronTriggerExpression: '@daily',
  ],
])
```